scripts/t2t

```diff
@@ -138,30 +142,30 @@ def validate(taxonomy, limit, flat_errors, hierarchy_errors):
 @click.option('--quiet', is_flag=True, default=False, help='Suppress output')
 def consistency(tree, consensus_map, output_file, rooted, quiet):
     """Consistency of a tree relative to taxonomy"""
-    
+
     if not quiet:
         print 'Determining taxonomic consistency of: '
```

Comment:
Non-blocking, but in hindsight, these print statements should be `click.echo` instead. Primary reason is that, if these interfaces end up using click's testrunner, then it can readily suppress the output.